src/schema/schema_create.c

```diff
@@ -1216,9 +1226,11 @@ __schema_create_config_check(
       (!tiered_name_set || !WT_STRING_MATCH("none", cval.str, cval.len));
 
     /* The import.file_metadata configuration is incompatible with tiered storage. */
-    if (is_tiered && file_metadata)
+    if (is_tiered && file_metadata) {
+        WT_STAT_CONN_INCR(session, session_table_create_import_fail);
```

Comment:
Why do we increment `session_table_create_import_fail` in some branches in the code but not in all of them? For instance, if `__wt_find_import_metadata` fails we probably should increment it too.

Reply:
This is why I think the top-level statistic only is a good idea because it keeps all these code paths clean and less prone to errors or missing the statistic. If later usage determines finer-grained information is necessary more/other statistics could be added. But for now I think we should keep it clean and simple and your comment is one reason why.